test/JDBC/input/ownership/BABEL-ROLE-vu-prepare.sql

```diff
@@ -0,0 +1,64 @@
+CREATE PROC test_role_proc_babel_user_ext_master AS
```

Comment:
we try to use filename as prefix to simplify having unique name across objects, babel-role-vu-prepare_user_ext_master or similar

Reply:
This is fixed in the latest commit